project/src/main/world/creature/emote-player.gd

```diff
@@ -394,12 +394,23 @@ green.
 """
 func _post_unemote() -> void:
 	for emote_sprite in _emote_sprites:
-		emote_sprite.scale = Vector2(2.0, 2.0)
-		emote_sprite.rotation_degrees = 0.0
+		emote_sprite.frame = 0
 		emote_sprite.modulate = Color.transparent
+		emote_sprite.rotation_degrees = 0.0
+		emote_sprite.scale = Vector2(2.0, 2.0)
+		emote_sprite.position = Vector2(0, 0)
+		if emote_sprite.material:
+			if emote_sprite.material.get("blend_mode"):
+				emote_sprite.material.blend_mode = SpatialMaterial.BLEND_MODE_MIX
+			if emote_sprite.material.has_method("set_shader_param") and Engine.is_editor_hint():
```

Comment:
Leave a comment describing why we only reset this in the editor.

Reply:
Added a comment, `In the editor, we reset shader parameters to known values ...`